src/components/ApplicationInsightService.tsx

```diff
@@ -0,0 +1,29 @@
+import { ApplicationInsights, ITelemetryItem } from '@microsoft/applicationinsights-web';
+import { ReactPlugin } from '@microsoft/applicationinsights-react-js';
+
+interface ApplicationInsightProps {
+  connectionString: string;
```

Comment:
We have two different entity in Application Insight one is **Instrumentation Key** and another one is **Connection String** , and as per official article from MSFT, they said to use Connection String only as they may stop using Instrumentation key in near future and if we use the name Instrumentation key instead of connection string that may cause confusion ,I have attached a screenshot of the same as well. kindly let me know your views on this.
![Application Insight](https://github.com/iTwin/synchronization-report-react/assets/110657554/c2b6f2cd-9ae9-46f9-9789-86edf0a0ffaa)

Reply:
ok, did not know that Instrumentation key is deprecated